webui-src/app/people/people_ownids.js

```diff
@@ -1,213 +1,399 @@
+      m("hr"),
+
+      m("input[type=password][placeholder=Passpharse]", {
+        style: "margin-top:50px;width:80%",
+        oninput: (e) => {
+          passphase = e.target.value;
+        },
+      }),
+      m(
+        "button",
+        {
+          style: "margin-top:160px;",
+          onclick: () => {
+            new Promise(function (resolve, reject) {
+              rs.rsJsonApiRequest("/rsIdentity/getOwnSignedIds", {}, (owns) => {
+                owns.ids.length > 0 ? resolve(owns.ids[0]) : resolve(id);
+              });
+            })
+              .then(function (result) {
+                return new Promise((resolve, reject) => {
+                  if (result === "") resolve(false);
+
+                  rs.rsJsonApiRequest(
+                    "/rsIdentity/createIdentity",
+                    {
+                      id: result,
+                      name: v.attrs.name,
+                      //avatar: v.attrs.details.mAvatar.mData.base64,
+                      pseudonimous: v.attrs.pseudonimous,
+                      pgpPassword: passphase,
+                    },
+                    (data) => {
+                      resolve(data.retval);
+                    }
+                  );
+                });
+              })
+              .then(function (result) {
+                const message = result
+                  ? "Successfully created identity."
+                  : "An error occured while creating identity.";
+                widget.popupMessage([
+                  m("h3", "Create new Identity"),
+                  m("hr"),
+                  message,
+                ]);
+              });
+          },
+        },
+        "Enter"
+      ),
+    ],
+  };
+};
 const CreateIdentity = () => {
   // TODO: set user avatar
-  let name = '',
-    pgpPassword = '',
+  let name = "",
+    id = "",
+    pgpPassword = "",
     pseudonimous = false;
   return {
-    view: () => [
-      m('i.fas.fa-user-plus'),
-      m('h3', 'Create new Identity'),
-      m('hr'),
-      m('input[type=text][placeholder=Name]', {
+    view: (v) => [
+      m("i.fas.fa-user-plus"),
+      m("h3", "Create new Identity"),
+      m("hr"),
+      m("input[type=text][placeholder=Name]", {
         value: name,
-        oninput: e => (name = e.target.value),
+        oninput: (e) => (name = e.target.value),
       }),
-      'Type:',
       m(
-        'select',
+        "div",
         {
-          value: pseudonimous,
-          oninput: e => (type = e.target.value),
+          style: "display:inline; margin-left:5px;",
         },
         [
-          m('option[value=false]', 'Linked to your Profile'),
-          m('option[value=true]', 'Pseudonymous'),
-        ],
+          "Type:",
+          m(
+            "select",
+            {
+              value: pseudonimous,
+              style: "border:1px solid black",
+              oninput: (e) => {
+                pseudonimous = e.target.value === "true" ? true : false;
```

Comment:
Sorry I totally didn't notice this :facepalm: , we could go with:
```suggestion
                pseudonimous = e.target.value === "true";
```
I feel stupid now haha.